core/services/pipeline/task.hexencode.go

```diff
@@ -0,0 +1,83 @@
+package pipeline
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+)
+
+//
+// Return types:
+//     string
+//
+type HexEncodeTask struct {
+	BaseTask `mapstructure:",squash"`
+	Input    string `json:"input"`
+}
+
+var _ Task = (*HexEncodeTask)(nil)
+
+func (t *HexEncodeTask) Type() TaskType {
+	return TaskTypeHexEncode
+}
+
+func addHexPrefix(val string) string {
```

Comment:
`utils.AddHexPrefix`

Reply:
This one will add "0x" even to an empty string - is that what we want? (see tests).

Reply:
Good point that I've missed. Then your approach is right. Thank you.